### DISCUSSION TITLE
console.cloud.google.com renders weirdly

### DISCUSSION THREAD

**Discussion by KaffeMyers** [qutebrowser/qutebrowser#7458] [Q&A] [answered]
I use Google Cloud at work and even if I spend most of my time in the CLI, it is nice to browse around the web GUI every once in a while for various reasons, and going forward I'll probably spend some time using their dashboards. However, I currently need to open up a separate Firefox browser for this as it appears as if qutebrowser can't render the cloud console properly. I of course would like to use qutebrowser for everything.

Is anyone else facing the same issues?

**Comment by The-Compiler** [comment, marked as answer]
It looks like they dropped compatibility with the older Chromium version Qt 5 is based on. Unfortunately, I wasn't able to find out what exactly the culprit is, so this is something we'll have to live with until the Qt 6 branch is ready...

You might want to test it already though, that should work fine (and it's generally very stable) - see #7202 for details.
